sdk-react/src/AnimaSdkProvider.tsx

```diff
@@ -130,6 +139,68 @@ export function AnimaSdkProvider({ figmaRestApi, f2cUrl, l2cUrl, p2cUrl, childre
     }
   };
 
+  const attachJob = async (sessionId: string, params: UseAnimaParams) => {
+    const storedType = localStorage.getItem(`anima:${sessionId}:type`);
```

Comment:
We should not enforce localStorage usage in a public SDK, this should be an argument instead.

Reply:
I think tthe caller from this method should send the job type